config/config.json

```diff
@@ -1,14 +1,14 @@
 {
   "development": {
     "username": "root",
-    "password": "root",
+    "password": "rootpass",
```

Comment:
We should probably keep this as root, i think that's what everyone else is using.